docs/src/humann.md

```diff
@@ -1,4 +1,613 @@
+red_201901b_subset.tar.gz
+uniref : uniref90_ec_filtered_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_ec_filtered/uniref90_ec_filte
+red_201901b_subset.tar.gz
+uniref : DEMO_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref90_DEMO_diamond_v201901b.tar.
+gz
+utility_mapping : full = http://huttenhower.sph.harvard.edu/humann_data/full_mapping_v201901b.tar.gz
+```
+
+For example, if you'd like to install these databases to `/BigDrive/humann/`,
+you could run
+
+```julia-repl
+shell> humann_databases --download cholophlan full /BigDrive/humann/chocophlan
+# ... lots of output
+
+shell> humann_databases --download uniref uniref90_diamond /BigDrive/humann/uniref
+# ... lots of output
+
+shell> humann_databases --download utility_mapping full /BigDrive/humann/utility_mapping
+# ... lots of output
+```
+
+At some point, I'll write some functions to automate this, but for now,
+doing this will update a configuration file, so you shouldn't have to worry about it again.
+
+## Running HUMAnN
+
+[Official tutorial link](https://github.com/biobakery/biobakery/wiki/humann3#2-metagenome-functional-profiling)
+
+Some example files you can use to run this tutorial are available from the MetaPhlAn repo,
+and can be downloaded using the `Downloads` standard library in julia:
+
+```julia-repl
+julia> using Downloads: download
+
+julia> base_url = "https://github.com/biobakery/humann/raw/master/examples/";
+
+julia> files = [
+           "demo.fastq.gz",
+           "demo.sam",
+           "demo.m8"
+       ];
+
+julia> for file in files
+           download(joinpath(base_url, file), file)
+       end
+
+julia> readdir()
+5-element Vector{String}:
+ "Manifest.toml"
+ "Project.toml"
+ "demo.fastq.gz"
+ "demo.sam"
+ "demo.m8"
+```
+
+For convenience, this package has the [`humann()`](@ref) function,
+which can be used in your julia scripts to build and call the `humann` command line tool.
+
+For example, rather than call
+
+```sh
+$ humann --input demo.fastq.gz --output demo_fastq
+```
+
+You can do
+
+```julia-repl
+julia> humann("demo.fastq.gz", "demo_fastq")
+[ Info: Running command: humann -i demo.fastq.gz -o demo_fastq
+Creating output directory: /home/kevin/my_project/demo_fastq
+Output files will be written to: /home/kevin/my_project/demo_fastq
+Decompressing gzipped file ...
+```
+
+First, `humann` will run `metaphlan` to generate taxonomic profiles,
+then will use that taxonomic profile to run a custom gene search.
+## Default outputs
+
+[Official tutorial link](https://github.com/biobakery/biobakery/wiki/humann3#23-humann-default-outputs)
+
+To load a profile generated by `humann`,
+use the [`humann_profile`](@ref) function:
+
+```julia-repl
+julia> gfs = humann_profile("demo_fastq/demo_genefamilies.tsv")
+Microbiome.CommunityProfile{Float64, Microbiome.GeneFunction, Microbiome.MicrobiomeSample} with 589 things in 1 places
```

Comment:
Should this `589 features in 1 samples`?

Also, when I test this function, I get `589 things in 1 places` and then `Thing names:` and `Place names`
I have the main branch of Microbiome added to my package REPL. What do I need to do to update it?

Reply:
`]up` 

This specific change actually comes from a dependency - EcoBase.jl, which needs to be on the most recent version. You can search your Manifest.toml to see what's installed.